qa/tasks/vstart_runner.py

```diff
@@ -381,7 +381,12 @@ def _do_run(self, args, check_status=True, wait=True, stdout=None,
                     args[0]
                 ))
 
-        log.info("Running {0}".format(args))
+        logmsg = '>'
+        for arg in args:
+            # we need to convert to string in case of args like Raw('&&')
+            logmsg += ' ' + (str(arg.value) if isinstance(arg, Raw) else arg)
+        log.info(logmsg)
+        del logmsg
```

Comment:
`del` does not actually delete anything, it just decrements ref counters... and you don't really need to worry about that (Python will delete when this function ends). If you dislike too much the idea of using an object for storing temporary data, and code readability is not compromised, you could try and inline everything together like:
```suggestion
        log.info('> %s', ' '.join(str(arg.value) if isinstance(arg, Raw) else arg for arg in args)
```

Reply:
Fixed, thanks!